cfgov/v1/util/ref.py

```diff
@@ -67,8 +67,8 @@
     )),
     ('Blog', (
         ('at-the-cfpb', 'At the CFPB'),
-        ('policy_compliance', 'Policy & Compliance'),
-        ('data-research-reports', 'Data, research & reports'),
+        ('policy_compliance', 'Policy and compliance'),
```

Comment:
It was this way before,  but would it be a pain to fix this so it's not the only item in this list with a slug that uses underscores instead of dashes?

Reply:
Yes, it would run into the same issue as noted above.

Reply:
Yes. This code is very, very sad but refactoring it is not easy. See also GHE/CFGOV/platform/issues/825